bin/hdfcoinc/pycbc_multiifo_coinc_statmap

```diff
@@ -0,0 +1,472 @@
+#!/usr/bin/env python
+"""
+The program combines coincident output files generated
+by pycbc_multiifo_coinc_findtrigs to generated a mapping between SNR and FAP, along
+with producing the combined foreground and background triggers. It also has
+the capability of doing hierarchical removal of foreground triggers that are
+louder than all of the background triggers. We use this to properly assess
+the FANs of any other gravitational waves in the dataset.
+"""
+import argparse, h5py, itertools
+import lal, logging, numpy
+from pycbc.events import veto, coinc
+import pycbc.version, pycbc.pnutils, pycbc.io
+import sys
+
+def sec_to_year(sec):
+    return sec / lal.YRJUL_SI
+
+class fw(object):
```

Comment:
Would prefer a more descriptive name for the class following pythons ClassNamingConvention.

Reply:
(If this is just copied from the old code then treat this as an optional comment).

Reply:
This is a copy from the old code, as well as the function before it. It is defined in the other statmap file, not sure if it exists anywhere else.